apps/web/src/utils/contractHelpers.ts

```diff
@@ -200,7 +200,7 @@ export const getPointCenterIfoContract = (signer?: Signer | Provider) => {
 export const getCakeContract = (signer?: Signer | Provider, chainId?: number) => {
   return getContract({
     abi: cakeAbi,
-    address: chainId ? CAKE[chainId].address : CAKE[ChainId.BSC].address,
+    address: chainId ? vertoTokens.verto.address : vertoTokensTestnet.verto.address,
```

Comment:
This should be be checking if its mainnet, not checking for a chainId, please look at other places in the code where we do this,I think its `IS_MAINNET` or something like that